integration-tests/utils/integration-utils.js

```diff
@@ -277,8 +277,16 @@ const logPageConsoleMessages = page => {
 }
 
 const logPageAjaxResponses = page => {
+  const terraRequests = [
+    'broad',
+    'terra'
+  ]
   const handle = res => {
-    rawConsole.log('page.http.res', `${res.status()} ${res.request().method()} ${res.url()}`)
+    const request = res.request()
+    if (terraRequests.some(urlPart => request.url().includes(urlPart))) {
+      console.log('page.http.res', `${request.method()}\t${res.status()}  ${res.url()}`)
+      // rawConsole.log('page.http.res', `${res.status()} ${res.request().method()} ${res.url()}`)
```

Comment:
This is the tradeoff. Jest fails to capture `rawConsole` logs so I can't use it anymore. Console logs in Terminal window will show stack trace, but logs in log file will not show any stack trace, just like rawConsole.

Reply:
Correct. I have not made any attempt to replace all `rawConsole`. I'd update all `rawConsole` in a separate follow-up PR after this PR.

Reply:
Alright, I will do that in this PR.